### PR TITLE
commit from Ammar

### PR DESCRIPTION
**Pull request by Ammars24**
my great commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # GitHub Study Project
 This is repository for researching GitHub tools and fuetures
+change from Ammar
```
Comment (rasnaut) [inline README.md:3]: Need Capital letter

**Review by rasnaut — CHANGES_REQUESTED** [review]
Change According to comment

File: README.md (hunk 1 above)
Comment (Ammars24) [reply README.md:3]: Do you think better with a capital letter

**Review by adragin — COMMENTED** [review]
all is ok

**Comment by rasnaut** [comment]
Comment

**Pull request merged**